app/build.gradle

```diff
@@ -17,6 +17,10 @@ android {
         testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
 
         multiDexEnabled true
+
+        jackOptions {
+            enabled true
```

Comment:
The jack toolchain is deprecated:
https://android-developers.googleblog.com/2017/03/future-of-java-8-language-feature.html

Reply:
Oh no.. closing then. Will wait for AS 3.0 to be stable.